spec/bolt_ext/server_spec.rb

```diff
@@ -38,7 +38,7 @@ def app
           'user' => target[:user],
           'password' => target[:password],
           'port' => target[:port],
-          'options' => { "host-key-check" => "false" }
```

Comment:
Is travis still failing with this fix?

Reply:
Yes, though it's passing locally for me....